### DISCUSSION TITLE
Route [orders.success] not defined.

### DISCUSSION THREAD

**Discussion by UnrulyNatives** [mnastalski/przelewy24-laravel#4] [General]
Cześć. 
Próbuję uruchomić płatność Przelewy24, ale jestem jedynie amatorem w programowaniu PHP. 
Czy moglibyśmy uzupełnić kilka usterek tego repo? 
Nie ma w nim kilku rzeczy, które trzeba odgadywać: 
- Brak pliku modelu
- Nazwy kolumn tabeli trzeba zgadywać z kodu kontrolera, co uczyniłem.
- Ale nie potrafię odgadnąć tego, jak zdefiniować route:
` Route [orders.success] not defined.`

Bardzo proszę o pomoc.
Na marginesie, w paracy opieram się na repo https://github.com/binafy/laravel-cart, które wygląda na obiecujące narzędzie pod względem prostoty i chęci autora do koooperacji.

**Comment by mnastalski** [comment]
Wszelkie parametry są całkowicie przykładowe, repo nie polega na i nie dostarcza konkretnych modeli czy innych struktur a ma jedynie ułatwiać implementację [mnastalski/przelewy24-php](https://github.com/mnastalski/przelewy24-php/).

Jeśli chodzi o niezdefiniowany route to musisz zdefinować nazwę route'a używając [named routes](https://laravel.com/docs/11.x/routing#named-routes).
